tests/test_evaluation.py

```diff
@@ -0,0 +1,40 @@
+import os
+import json
+from src import evaluation
+
+CURRENT_PATH = os.path.dirname(__file__)
+
+
+def test_generate_result_success():
+    fake_report = os.path.join(CURRENT_PATH, 'fixture', 'report.json')
+
+    req_mock = os.path.join(CURRENT_PATH, 'fixture', 'requirements.json')
+    reqs = load_file(req_mock)
+    requirements = list(reqs['requirements'])
+
+    result = evaluation.generate_result(fake_report, req_mock)
+
+    assert 'evaluations' in result
+    evaluations = list(result['evaluations'])
+    assert len(evaluations) == len(requirements)
+    # If test passed, should return passed grade
+    assert evaluations[0]['grade'] == evaluation.PASSED_GRADE
+    assert evaluations[0]['description'] == requirements[0]['description']
+    # If test failed, should return failed grade
+    assert evaluations[1]['grade'] == evaluation.FAILED_GRADE
+    assert evaluations[1]['description'] == requirements[1]['description']
+    # If test is not found on report, should return failed grade
+    assert evaluations[2]['grade'] == evaluation.FAILED_GRADE
+    assert evaluations[2]['description'] == requirements[2]['description']
+
+
+def test_enums():
+    evaluation.FAILED_GRADE == 1
+    evaluation.PASSED_GRADE == 3
```

Comment:
Dessa forma os testes sempre passarão para essa função, é preciso do `assert`.
Você pode testar se fazer `evaluation.FAILED_GRADE == 'foo'` e escrever no terminal `make test`, que vai passar, sendo que não deveria. Logo, você poderia mudar para o seguinte 🙂 :
```suggestion
    assert evaluation.FAILED_GRADE == 1
    assert evaluation.PASSED_GRADE == 3
```

Reply:
Realmente esqueci disso hehehe

Reply:
De boas mano 🙂